routes/web.php

```diff
@@ -36,6 +37,20 @@
 Route::get('/lists/tools', function () {
     return view('lists.tools');
 })->name('lists.tools');
+
 Route::get('/songs', function () {
     return view('songs.index');
 })->name('songs.index');
+Route::get('/songs/show', function () {
+    return view('songs.show');
+})->name('songs.show');
+Route::get('/songs/edit', function () {
+    return view('songs.edit');
+})->name('songs.edit');
+Route::get('/songs/create', function () {
+    return view('songs.create');
+})->name('songs.create');
```

Comment:
Sooner or later you are going to need a songController

Reply:
Today I added M and C to the MVC. The controllers are almost here

Reply:
Nice!